test/Main.hs

```diff
@@ -1,20 +1,21 @@
-import Algebra.Graph.Test.Acyclic.AdjacencyMap
-import Algebra.Graph.Test.AdjacencyIntMap
-import Algebra.Graph.Test.AdjacencyMap
-import Algebra.Graph.Test.NonEmpty.AdjacencyMap
-import Algebra.Graph.Test.Export
-import Algebra.Graph.Test.Graph
-import Algebra.Graph.Test.NonEmpty.Graph
-import Algebra.Graph.Test.Internal
-import Algebra.Graph.Test.Labelled.AdjacencyMap
-import Algebra.Graph.Test.Labelled.Graph
-import Algebra.Graph.Test.Relation
-import Algebra.Graph.Test.Relation.SymmetricRelation
-import Algebra.Graph.Test.Bipartite.AdjacencyMap
-import Data.Graph.Test.Typed
+import           Algebra.Graph.Test.Acyclic.AdjacencyMap
+import           Algebra.Graph.Test.AdjacencyIntMap
+import           Algebra.Graph.Test.AdjacencyMap
+import           Algebra.Graph.Test.Bipartite.AdjacencyMap
+import           Algebra.Graph.Test.Export
+import           Algebra.Graph.Test.Graph
+import           Algebra.Graph.Test.Internal
+import           Algebra.Graph.Test.Labelled.AdjacencyMap
+import           Algebra.Graph.Test.Labelled.Graph
+import           Algebra.Graph.Test.NonEmpty.AdjacencyMap
+import           Algebra.Graph.Test.NonEmpty.Graph
+import           Algebra.Graph.Test.Relation
+import           Algebra.Graph.Test.Relation.SymmetricRelation
+import           Algebra.Graph.Test.Undirected
+import           Data.Graph.Test.Typed
```

Comment:
These changes seem accidental, please undo them. Same for the two imports below.

Reply:
It is! I'm sorry about this, my Haskell setup was doing this automatically. I already disabled it!